src/qt/dash.cpp

```diff
@@ -775,6 +775,11 @@ int main(int argc, char *argv[])
 
         GUIUtil::setStyleSheetDirectory(strCustomDir);
     }
+    // Validate -debug-ui
+    if (gArgs.IsArgSet("-debug-ui")) {
```

Comment:
Should use `GetBoolArg` for `-debug-ui` here and in all other cases, basically s/`IsArgSet("-debug-ui")`/`GetBoolArg("-debug-ui", false)`/